test/v3/addLiquidityNested/addLiquidityNestedV3.integration.test.ts

```diff
@@ -53,7 +53,7 @@ const usdcToken = new Token(chainId, USDC.address, USDC.decimals);
 const wethToken = new Token(chainId, WETH.address, WETH.decimals);
 const mainTokens = [wethToken, daiToken, usdcToken];
 
-describe.skip('V3 add liquidity nested test, with Permit2 direct approval', () => {
+describe('V3 add liquidity nested test, with Permit2 direct approval', () => {
```

Comment:
These skips are handled in my recent PR so you can just remove this diff to avoid conflicts.

Reply:
removed the dif.